doc/semitrans.xml

```diff
@@ -376,3 +376,19 @@ gap> Size(WW);
     </Description>
   </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="PartitionMonoidSmallGeneratingSet">
+   <ManSection>
+    <Oper Name="SmallGeneratingSet" Arg="G"/>
+    <Returns>A small generating set with maximal idempotent elements.</Returns>
+    <Description>
+      When <A>G</A> is a subgroup of the Partition Monoid on <A>n</A> points. 
```

Comment:
I think this is a confusing way to word the type of argument this function works on. You explained earlier that, actually, as long as `G` is a semigroup that contains the singular partition monoid on `n` points as well as at least one permutation, that fits the bill. Maybe something like this would be clearer:
```
If <A>G</A> is a bipartition semigroup on <M>n</M> points
which contains every element of <C>SingularPartitionMonoid(n)</C>,
along with at least one permutation, then this function returns...
```
Remember that you should be using:
- `<A>...</A>` tags for **arguments**, i.e. in this case only `G`
- `<C>...</C>` tags for **code**, such as `SingularPartitionMonoid(n)`
- `<M>...</M>` tags for **mathematical expressions**, including, I would say, `n`.